scylla-cdc-replicator/src/replicator_consumer.rs

```diff
@@ -734,8 +750,8 @@ impl ReplicatorConsumer {
         values_for_update.extend(values.iter());
 
         let mut values_for_list_update = Vec::with_capacity(3 + values.len());
-        values_for_list_update.extend([Some(&ttl), None, None]);
-        values_for_list_update.extend(values.iter());
+        values_for_list_update.extend([Some(ttl.clone()), None, None]);
+        values_for_list_update.extend(values.iter().map(|o| o.cloned()));
```

Comment:
Ah, dang it, so now we _always_ clone all values here, even if there are no lists in the table... Now I understand that this is a tradeoff. I think the previous version was better after all.

I think we could have our cake and eat it if `SerializeValue` were implemented for `Cow`, then we could avoid copying all values in all cases, only the timeuuid value would have to be copied as uuid. However, we don't have this, and I don't think it's worth inventing something here or waiting for the driver to implement - so, please switch to the previous version as I think it was the lesser evil.

As a side note, I don't really like that `values_for_list_update` is created even if there are no lists, but this is a preexisting problem.